csrc/kernels.hip

```diff
@@ -2853,6 +2853,7 @@ template <int THREADS, int ITEMS_PER_THREAD, int TILE_ROWS, int TILE_COLS, int T
 #define DENORM 1.0f/127.0f
 #define MAX_SPARSE_COUNT 32
 #define SMEM_SIZE 8*256
+#define WARP_SIZE __AMDGCN_WAVEFRONT_SIZE
```

Comment:
Instead of __AMDGCN_WAVEFRONT_SIZE can we use the warpSize defined in hip/hip_runtime.h header file. 

#define WARP_SIZE warpSize
For example: https://github.com/pytorch/pytorch/blob/2cefbb71cf65696e28ee1bdfce06d6846285e5fb/c10/macros/Macros.h#L322

Reply:
Changed it. I have modified 4 bit gemm kernel also to use the same.